lib/dealership.rb

```diff
@@ -0,0 +1,86 @@
+class Dealership
+
+  attr_reader :name, :address, :inventory
+
+  def initialize(name, address)
+    @name = name
+    @address = address
+    @inventory = []
+  end
+
+  def inventory_count
+    @inventory.length
+  end
+
+  def has_inventory?
+    @inventory.length >= 1
+  end
+
+  def add_car(car)
+    @inventory << car
+  end
+
+  def cars_by_make(make)
+    cars_by_make_catcher = []
+    @inventory.each do |car|
+      if car.make == make
+        cars_by_make_catcher << car
+      end
+    end
+    cars_by_make_catcher
+  end
```

Comment:
Yes, totally! Another way you could also do this is with the `find_all` enumerator. 

```
def cars_by_make(make)
  @inventory.find_all do |car|
    car.make == make
  end
end
```